schema/schema2/typesystem_test.go

```diff
@@ -1,212 +0,0 @@
-package schema
-
-import (
-	"fmt"
-	"strings"
-	"testing"
-
-	. "github.com/warpfork/go-wish"
-
-	"github.com/ipld/go-ipld-prime/codec/dagjson"
-	schemadmt "github.com/ipld/go-ipld-prime/schema/dmt"
-)
-
-func TestBuildTypeSystem(t *testing.T) {
```

Comment:
Similar to [above](https://github.com/ipld/go-ipld-prime/pull/245/files#r704275918), I'd like to flag these tests as one of few things still worth dredging out of here.

There's not much to them, but it's some of the only tests we've got so far which test the logical validation rules.  (Also, test that things can flunk them!)  Even if the coverage of these is skeletal so far, it would be nice to make sure we know where we're going to put things like this.

(Or, better yet, if we can port them to data-driven stuff now, and maybe put them in the ipld/ipld meta repo, that'd be..... really awesome.)